homeassistant/components/frontend/www_static/home-assistant-polymer

```diff
@@ -1 +1 @@
-Subproject commit 1ad673ff6f946de2a6358efcdf59667fbc73a05a
+Subproject commit a21547abfb8d19605850ec4bee48f837d607f736
```

Comment:
No, because it is needed for when we add a new compiled version of the frontend to the main repo.